invenio_records_rest/views.py

```diff
@@ -257,7 +257,7 @@ def links_factory(pid, record=None, **kwargs):
         search_class, default=RecordsSearch
     )
     indexer_class = obj_or_import_string(
-        indexer_class, default=RecordIndexer
+        indexer_class, default=None
```

Comment:
Is this because you have problems in with deposit? It ought to have set this to None in Invenio-Deposit.

Reply:
Ah, is not fixed in Invenio-Deposit....so I think this fix should be applied to Invenio-Deposit instead....I'll pass by the office.

Reply:
@lnielsen I think this is a mistake I did, we had set it to None when we were using the object instance `indexer` and testing with invenio-deposit to avoid the double indexing, but I refactored it and set the default to RecordIndexer.

Reply:
We decided to isolate it here, setting it to none by default, so no new PR on Deposit needed to be done back then

Reply:
But shouldn't we enable indexing by default? and make fix to Invenio-Deposit?